### ISSUE TITLE
Add possibility to calculate FocalLoss from output of the sigmoid. 

### ISSUE THREAD

**Issue by razmikmelikbekyan** [qubvel/segmentation_models.pytorch#588]
Currently Focal Loss supports the logits only as an input, but the others - Dice and Jacard are supporting the probabilities as an input (`from_logits=False`). It would nice if the focal will have the same interface.
